app/components/login/Login.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { createClass, Component } from 'react';
+
+const Login = createClass({
+
+  signUserIn(email, password) {
```

Comment:
Thank you so much for your insight as we try to rap our heads around this. Do you have any resources for react-router-dom 4.  We are having a tough time figuring that out.